.github/workflows/basic-ci-amd64.yaml

```diff
@@ -56,6 +56,50 @@ jobs:
         timeout-minutes: 10
         run: bash tests/integration/cri-containerd/gha-run.sh run
 
+jobs:
```

Comment:
I don't think this line is needed/valid? `jobs:` is already defined above on line 16?

Reply:
Fixed